### PR TITLE
Added Examples for Validating Dynamic Content to the Model Validation…

### LINKED ISSUE #4752: Explain How to Add Client Side Validation to Dynamic Forms
Dynamically generated content does not exhibit client side validation automatically.  Getting this to work correctly requires calling a `parse()` method in the jQuery Unobtrusive Validation library that is not documented anywhere (I discovered it from [this](https://stackoverflow.com/a/14982495/3991688) SO post).  I imagine this is a pretty common use case, so it really should be described in the official docs somewhere (even if it feels kind of like a hack...).

## Suggested Reviewers:
@rachelappel 
@Rick-Anderson 
@isaac2004 
@guardrex 
@v-anpasi

### PR DESCRIPTION
**Pull request by Rabadash8820**
Fixes #4752

## Suggested Reviewers:
@rachelappel 
@Rick-Anderson 
@isaac2004 
@guardrex 
@v-anpasi

### CODE REVIEW

File: aspnetcore/mvc/models/validation.md [hunk 1]
```diff
@@ -144,6 +144,50 @@ Client-side validation prevents submission until the form is valid. The Submit b
 
 MVC determines type attribute values based on the .NET data type of a property, possibly overridden using `[DataType]` attributes. The base `[DataType]` attribute does no real server-side validation. Browsers choose their own error messages and display those errors however they wish, however the jQuery Validation Unobtrusive package can override the messages and display them consistently with others. This happens most obviously when users apply `[DataType]` subclasses such as `[EmailAddress]`.
 
+### Adding Validation to Dynamic Forms:
+
+Because jQuery Unobtrusive Validation passes validation logic and parameters to jQuery Validate when the page first loads, dynamically generated forms will not automatically exhibit validation. Instead, you must tell jQuery Unobtrusive Validation to parse the dynamic form immediately after creating it. For example, the code below shows how you might set up client side validation on a form added via AJAX.
+
+```js
```
Comment (Rabadash8820) [inline aspnetcore/mvc/models/validation.md]: Wasn't sure if this is the correct syntax for JavaScript code snippets

**Comment by Rick-Anderson** [comment]
@Rabadash8820  any reason these changes are not in #4751?

**Comment by Rabadash8820** [comment]
My original focus with #4751 was the quote from the jQuery Validate docs in the remote validation section.  This new blurb about validating dynamic forms seemed distinct enough to warrant it's own issue.  However, they do both kinda fall under the umbrella of discussing jQuery validate more, so I could cherry-pick these changes over to that branch if that would be more appropriate.

**Comment by Rick-Anderson** [comment]
@Rabadash8820  that's fine, no need to change anything. In the future keep in mind you can always close 2 issues with 1 PR.  We usually like to group changes in one file in one PR.

Thanks for helping out.

**Comment by Rick-Anderson** [comment]
@Rabadash8820 please review (and his other PR)

**Comment by Rabadash8820** [comment]
@Rick-Anderson  did you mean to mention @rachelappel?

**Comment by Rick-Anderson** [comment]
@rachelappel  please review (and the other PR)
@Rabadash8820  Thanks!

**Comment by Rick-Anderson** [comment]
@Rabadash8820 Thanks for your contribution.

**Pull request merged**